packages/checkout/sdk/jest.config.ts

```diff
@@ -12,7 +12,7 @@ const config: Config = {
     '@imtbl/blockchain-data': '<rootDir>../../blockchain-data/sdk/src',
     '@imtbl/generated-clients': '<rootDir>../../internal/generated-clients/src'
   },
-  testEnvironment: 'node',
```

Comment:
needed to make the `window.navigator` works -- this is a must to correctly run the tests

Reply:
Can you not just add `/** @jest-environment jsdom */` to the top of the file instead of changing this for all of the tests. I think we have done this in other files as well to support jsdom testEnvironment.